.checkpatch.conf

```diff
@@ -24,4 +24,5 @@
 --ignore UNDOCUMENTED_DT_STRING
 --ignore DT_SPLIT_BINDING_PATCH
 --ignore DT_SCHEMA_BINDING_PATCH
+--ignore TRAILING_SEMICOLON
```

Comment:
This has been seen in other PRs. I'd push it in a dedicated PR for faster merge.